scripts/build/linux.sh

```diff
@@ -156,6 +156,12 @@ function installer {
   app_dir_create $source_directory $architecture $appdir_temporary_location
   rm -f $output_file
   ./scripts/build/AppImages/AppImageAssistant-$architecture $appdir_temporary_location $output_file
+
+  pushd $output_directory
+  zip Etcher-linux-$architecture.zip Etcher-linux-$architecture.AppImage
```

Comment:
@dlech Yeah, it will be added in the following PR (https://github.com/resin-io/etcher/pull/764#issuecomment-255871914), which will be rebased with these changes.